src/Sulu/Bundle/AdminBundle/Resources/js/containers/Form/fields/ResourceLocator.js

```diff
@@ -49,38 +51,15 @@ class ResourceLocator extends React.Component<FieldTypeProps<?string>> {
                 return;
             }
 
-            const {tags: finishedFieldTags} = formInspector.getSchemaEntryByPath(finishedFieldSchemaPath) || {};
-            if (!finishedFieldTags || !finishedFieldTags.some((tag) => tag.name === PART_TAG)) {
-                return;
-            }
 
-            const partEntries = formInspector.getPathsByTag(PART_TAG)
-                .map((path: string) => [path, formInspector.getValueByPath(path)])
-                .filter(([, value: mixed]) => !!value)
-                .map(([path: string, value: mixed]) => {
-                    // path is a jsonpointer but the api controller requires property names
-                    if (path.startsWith('/')) {
-                        return [path.substr(1), value];
-                    }
-
-                    return [path, value];
-                });
-
-            if (partEntries.length === 0) {
+            const {tags: finishedFieldTags} = formInspector.getSchemaEntryByPath(finishedFieldSchemaPath) || {};
+            if (
```

Comment:
Can you format it the same way it was before? Then it is easier to review for me :slightly_smiling_face: